base/pcre.jl

```diff
@@ -48,45 +48,72 @@ end
 
 # supported options for different use cases
 
+# arguments to pcre2_compile
 const COMPILE_MASK      =
+      ALT_BSUX          |
+      ALT_CIRCUMFLEX    |
+      ALT_VERBNAMES     |
       ANCHORED          |
+      # AUTO_CALLOUT    |
       CASELESS          |
       DOLLAR_ENDONLY    |
       DOTALL            |
+      # DUPNAMES        |
       ENDANCHORED       |
       EXTENDED          |
+      EXTENDED_MORE     |
       FIRSTLINE         |
+      LITERAL           |
+      MATCH_INVALID_UTF |
+      MATCH_UNSET_BACKREF |
       MULTILINE         |
-      NEWLINE_ANY       |
-      NEWLINE_ANYCRLF   |
-      NEWLINE_CR        |
-      NEWLINE_CRLF      |
-      NEWLINE_LF        |
+      NEVER_BACKSLASH_C |
+      NEVER_UCP         |
+      NEVER_UTF         |
       NO_AUTO_CAPTURE   |
+      NO_AUTO_POSSESS   |
+      NO_DOTSTAR_ANCHOR |
       NO_START_OPTIMIZE |
       NO_UTF_CHECK      |
+      UCP               |
       UNGREEDY          |
-      UTF               |
-      UCP
-
+      USE_OFFSET_LIMIT  |
+      UTF
+
+# arguments to pcre2_set_newline
+const COMPILE_NEWLINE_MASK = (
```

Comment:
What is this for?

Reply:
Seemed like someone wanted them in a list, so I kept them here